components/ws-proxy/pkg/proxy/routes.go

```diff
@@ -79,9 +79,10 @@ func installWorkspaceRoutes(r *mux.Router, config *RouteHandlerConfig, ip Worksp
 	// The favicon warants special handling, because we pull that from the supervisor frontend
 	// rather than the IDE.
 	faviconRouter := r.Path("/favicon.ico").Subrouter()
+	faviconPath := "/_supervisor/frontend/favicon.ico"
```

Comment:
can we avoid this file change, if its not doing anything functional

Reply:
That file path is used in browser tab icons. So you can configure it to your needs to change that icon.